build_tools/detail/linux_portable_build_in_container.sh

```diff
@@ -5,6 +5,8 @@ set -e
 set -o pipefail
 trap 'kill -TERM 0' INT
 
+pip install -r /therock/src/requirements.txt
```

Comment:
Probably want to defer any `pip` commands until _after_ setting `PIP_CACHE_DIR` (move this down a few lines)